packages/dara-core/dara/core/main.py

```diff
@@ -228,6 +230,14 @@ async def catchall_middleware(req: Request, call_next):
     for kind, handler in config.ws_handlers.items():
         custom_ws_handlers_registry.register(kind, handler)
 
+    # update encoder registry
+    for typ, encoder in config.encoders.items():
+        encoder_registry[typ] = encoder
```

Comment:
Just use .update(). It's faster than a loop.
```suggestion
encoder_registry.update(config.encoders)
```

Reply:
Thanks Roman, will address that. May I ask why the dict.update() is faster?

Reply:
I would guess it's simply because Python can internally optimize the method as compared to a manual loop.